### PR TITLE
doc: update the readme

### PR DESCRIPTION
**Pull request by yorkie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,62 +34,77 @@ push the whole area forward with the machine learning engineering. For this reas
...
+$ pipcook init --tuna
 ```
 
-Occuring the download problems? We use [tuna](https://mirror.tuna.tsinghua.edu.cn/help/pypi/) mirror to address this issue:
+Then run a pipeline:
 
-```sh
-$ pipcook init --tuna
+```shell
+$ pipcook run https://raw.githubusercontent.com/alibaba/pipcook/master/example/pipelines/text-bayes-classification.json
```
Comment (utkobe) [inline README.md:99]: Suggest to give out --verbose version of this command.  Probably also give another command for --tuna args.

Comment (yorkie) [reply README.md:99]: Should be the `--verbose` option set by default?

Comment (rickycao-qy) [reply README.md:99]: I think so for new users

Comment (yorkie) [reply README.md:99]: So shall we keep this here and change it at CLI?

Comment (rickycao-qy) [reply README.md:99]: Good idea

**Pull request merged**